src/components/selection/SelectionTypes.tsx

```diff
@@ -1,6 +1,13 @@
 enum SelectionTypes {
   CHECKBOX = 'checkbox',
+  SWITCH = 'switch',
   RADIO = 'radio',
 }
 
+export const SelectionInputTypes = {
+  [SelectionTypes.CHECKBOX]: 'checkbox',
+  [SelectionTypes.SWITCH]: 'checkbox',
```

Comment:
= 'switch' ?

Reply:
@AliciaSymphony 

This is the value that is injected in the `type` attribute of the `input`. `switch` is not a valid HTML input type.
It needs to be 'checkbox', or one of [https://www.w3schools.com/html/html_form_input_types.asp](https://www.w3schools.com/html/html_form_input_types.asp). 
<img width="282" alt="Screenshot 2021-04-16 at 10 36 24" src="https://user-images.githubusercontent.com/66251236/114997078-c65e7580-9e9f-11eb-8652-0eb65548eaf4.png">

Reply:
alright perfect, I was not sure if it was a typo or on purpose :)